### PR TITLE
homework2, changed variables eventCode & time, added variable price

### PR DESCRIPTION
**Pull request by MaVaYuran**

### CODE REVIEW

**Comment by MaVaYuran** [comment]
Sorry i forgot rename tickets

**Review by Alexmansar — APPROVED** [review]

**Comment by MaVaYuran** [comment]
I don't know ID correct now or not

File: src/main/java/homework2/Ticket.java [hunk 1]
```diff
@@ -1,48 +1,48 @@
 package homework2;
 
-import java.util.Date;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.Random;
 
 public class Ticket {
-    private String ID;
```
Comment (Alexmansar) [inline src/main/java/homework2/Ticket.java:7]: Please, leave string. It more suitable in this case) and just simple length validation

**Review by Alexmansar — CHANGES_REQUESTED** [review]

File: src/main/java/homework2/Ticket.java (hunk 1 above)
Comment (MaVaYuran) [reply src/main/java/homework2/Ticket.java:7]: ok

**Review by Alexmansar — APPROVED** [review]
@MaVaYuran Good job, please merge it to main))

**Pull request merged**